db/db_compaction_filter_test.cc

```diff
@@ -308,7 +308,7 @@ TEST_F(DBTestCompactionFilter, CompactionFilter) {
     ASSERT_OK(iter->status());
     while (iter->Valid()) {
       ParsedInternalKey ikey(Slice(), 0, kTypeValue);
-      ikey.sequence = -1;
+      ikey.sequence = SequenceNumber(-1);
```

Comment:
SequenceNumber is alias of uint64_t. Seems to me we can simply remove this line.